table-api/src/main/java/io/deephaven/api/TableOperations.java

```diff
@@ -166,6 +166,12 @@
 
     // -------------------------------------------------------------------------------------------
 
+    TOPS lazyUpdate(String... columns);
```

Comment:
This reminds me that JavaDoc should really make its way up to `TableOperations`, but not necessarily in this PR. I think `table-api` will never become a tool for new users without that.